plugins/spy.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: spy
+spec:
+  homepage: https://github.com/huazhihao/kubespy
+  shortDescription: pod debugging tool for kubernetes clusters with docker runtimes
+  version: v0.3.1
+  description: |
+    `kubespy` is a kubectl plugin implemented in bash to debug a application pod during runtime by creating and running an temporary `spy container` to join its docker namespace(eg. pid/net/ipc). You can specify the image of `spy container` which should include all the debug tools required and can load them into the debugging environment, so you don't have to unnecessarily bundle those debug tools with the application pod image.
+
+    Unlike [kubectl-debug](https://github.com/verb/kubectl-debug), `kubespy` doesn't require the prerequisites of 1. `EphemeralContainers` to be enabled in the cluster 2. `shareProcessNamespace` to be enabled for the application pod. The reasons are 1. `EphemeralContainers` is still in early alpha state and is not suitable for production clusters 2. modifying the spec of `shareProcessNamespace` will destroy the original application pod as well as the evidences inside.
```

Comment:
The explanation for 2 is not straightforward to understand, because you are talking about the situation where the config option was _not_ active when the error was encountered, and activating the option would re-create the pod. Maybe just skip the explanations altogether, WDYT?

You could also say something like

> `kubespy` is similar to [kubectl-debug](...). In contrast to the latter, kubespy works without the `EphemeralContainers` feature with is an experimental alpha feature and needs to be activated per pod.

Reply:
Agreed. Maybe I will just put 2 in the readme file of the plugin repo.